applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -65,7 +67,16 @@ struct sw_codec_config {
 	bool initialized;		 /* Status of codec */
 };
 
-/**@brief	Encode PCM data and output encoded data
+/**
+ * @brief	Check if sw_codec is initialized
+ *
+ * @retval	true	SW codec is initialized.
+ * @retval	false	SW codec is not initialized.
+ */
+bool sw_codec_is_initialized(void);
+
+/**
+ * @brief	Encode PCM data and output encoded data
```

Comment:
```suggestion
 * @brief	Encode PCM data and output encoded data.
```
Periods are missing at the end of the API documentation sentences throughout the file. Fix globally please.